src/main.cpp

```diff
@@ -69,15 +72,34 @@ void setup() {
   Serial.println("Done. RUNNING!");
   digitalWrite(LED_PIN, LOW);
 
+#define N 2
+#if N == 2
+  motor.move(360);
+#endif
```

Comment:
It would be neat if you could hook this up to the commander interface. I wont block PR on this, but if you wanted to make that happen instead of using preprocessor directives it would be nice.

Reply:
I try to test motor commands directly, without the commander interface. Not successful. :(